Source/WebCore/dom/TrustedType.h

```diff
@@ -78,4 +80,6 @@ ExceptionOr<bool> canCompile(ScriptExecutionContext&, JSC::CompilationType, Stri
 
 bool isEventHandlerAttribute(const QualifiedName& attributeName);
 
+ExceptionOr<String> trustedTypesCompliantAttributeValue(ScriptExecutionContext&, const String attributeType, const TrustedTypeOrString& value, String sink);
```

Comment:
Three ideas:
1) Don’t use `const String`, that makes no sense as an argument type.
2) Take `const String&` instead of `String`, no need to create reference count churn.
3) Consider returning at `AtomString` since that’s what all the callers need since that’s how we store attribute values.
```suggestion
ExceptionOr<AtomString> trustedTypesCompliantAttributeValue(ScriptExecutionContext&, const String& attributeType, const TrustedTypeOrString& value, const String& sink);
```